xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -645,6 +653,10 @@ bool CSystemGUIInfo::GetBool(bool& value, const CGUIListItem *gitem, int context
       }
       break;
     }
+    case SYSTEM_HAS_BACKLIGHT:
+      std::shared_ptr<IBacklight> backlight = CServiceBroker::GetBacklight();
+      value = !!backlight;
```

Comment:
maybe `static_cast<bool>(backlight)` to be a little more explicit about what is going on here?